src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -129,6 +135,8 @@ public Exporter(final Config config, final FcrepoClient.FcrepoClientBuilder clie
                 bagdir.mkdirs();
                 this.bag = new Bag(new Version(0, 97));
                 this.bag.setRootDir(bagdir);
+                this.bag.getMetadata().putAll(bagConfig.getBagInfo());
```

Comment:
@dbernstein: I've created a new PR that adds a fix for this to your commits: https://github.com/fcrepo4-labs/fcrepo-import-export/pull/66